docs/logging.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[logging-integrations]]
+=== Logging Integrations
+
+`elasticapm` provides integrations with both the default python logging library,
+as well as http://www.structlog.org/en/stable/[`structlog`].
+
+[[logging]]
+==== `logging`
+
+We provide a https://docs.python.org/3/library/logging.html#filter-objects[filter]
+which will add two new attributes to any filtered `LogRecord`:
+
+* `elasticapm_transaction_id`
```

Comment:
We should be using these fields: https://www.elastic.co/guide/en/ecs/current/ecs-tracing.html
There's also another field that's not part of ECS: `span.id`.

It should go like this:

- if there's an active span, then yield `span.id` and `trace.id`
- else if there's an active transaction, then yield `transaction.id` and `trace.id`

I think it would be helpful to provide a function to obtain the current trace IDs, so users don't have to implement this logic. This discussion over in the Node.js agent is relevant: https://github.com/elastic/apm-agent-nodejs/pull/1335#discussion_r322518554

Reply:
@axw side note on the naming: these are attributes on an object, so we can't use `.` in the name. The idea is to then use these attributes further down the line to include them in your log output, something like `trace.id=%(elasticapm_trace_id)s`

Reply:
I did provide public API entries for both of these IDs. I didn't spot `span.id` but I'll add it as well, and simplify the names. It will have to be `trace_id` and `transaction_id` as @beniwohli mentioned, and I'll still prepend `elasticapm_` because we're silently adding attributes and should namespace them as such.

Reply:
Done.

Reply:
Ahh I see, sorry. Perhaps it would be useful to add a fourth field then, which is a dict with the ECS field names? i.e. something like:

```python
elasticapm_labels = {
    'trace.id': '...',
    'transaction.id': '...',
    'span.id': '...',
}
```

Reply:
My only concern with that format is how difficult it will make using those attributes in a [logging Formatter](https://docs.python.org/3/library/logging.html#formatter-objects). I'll have to do a little investigation.

Reply:
Since the formatters just use simple printf-style string formatting, I don't think it's a good idea to nest all of these attributes. So, I did both. That should provide ease of access for users wanting to format the transaction into normal log messages, and consistency for those logging to JSON.